### PR TITLE
[SER-83] Set-up of database and the different entities used

### PR DESCRIPTION
**Pull request by Gugandeep**
Added the initial database set-up with GORM

### CODE REVIEW

File: serve-api/internal/db/db.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"gorm.io/gorm"
+)
+
+func Connect() *gorm.DB {
+	db_host := os.Getenv("POSTGRES_HOST")
+	db_user := os.Getenv("POSTGRES_USER")
+	db_pswd := os.Getenv("POSTGRES_PASSWORD")
+	db_name := os.Getenv("POSTGRES_DB")
+
+	dsn := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=5432 sslmode=disable", db_host, db_user, db_pswd, db_name)
```
Comment (mattmachell) [inline serve-api/internal/db/db.go:19]: You may need to parameterise sslmode as well for the move to production.

File: serve-web/src/Entity/Order.php [hunk 2]
```diff
@@ -158,7 +158,7 @@ abstract public function isOrderValid();
     /**
      * @var string|null
      *
-     * @ORM\Column(name="order_number", type="string", nullable=true, unique=true)
```
Comment (mattmachell) [inline serve-web/src/Entity/Order.php:161]: Was this a deliberate change in the PHP app?

Comment (coderste) [reply serve-web/src/Entity/Order.php:161]: I wasn't here when this change was made so @Gugandeep or @acsauk able to shine some light on this for us?

File: serve-api/internal/db/db.go (hunk 1 above)
Comment (acsauk) [reply serve-api/internal/db/db.go:19]: We'll pick this up as part of the ECS ticket seeing as the code we have right now isn't being built.

File: serve-web/src/Entity/Order.php (hunk 2 above)
Comment (acsauk) [reply serve-web/src/Entity/Order.php]: Reverted this - can't see why the change was made either.

**Review by GemTay — APPROVED** [review]

**Pull request merged**